gateway_test.go

```diff
@@ -843,6 +846,9 @@ func testHttp(t *testing.T, tests []tykHttpTest, separateControlPort bool) {
 			config.HttpServerOptions.OverrideDefaults = false
 		}
 
+		// Ensure that no local API's installed
+		os.RemoveAll(config.AppPath)
+		config.AppPath, _ = ioutil.TempDir("", "tyk-test-")
```

Comment:
This tests need to ensure that NO apis exists at start of the tests. So it removes all the previous API's before each test run. `os.RevemoAll` also removes the test directory, so I have to re-create it.